Core1Platform/03_InnovativeArchitecture.md

```diff
@@ -0,0 +1,29 @@
+# Innovative MACH+ architecture
+
+The MACH+ approach is considered as fundamental for the newest generation of software and systems integrators to quickly adapt new evolutions, trends, and technologies. The *Actindo Core1 Platform* offers maximum flexibility, highest performance, and future proofing by using the four guiding principles of microservices, API-first, cloud-native, and headless. 
+
+
+### Microservices
+
+With the help of pre-built commerce building blocks, your teams can easily create a customer-specific infrastructure. Furthermore, the teams can agilely scale and evolve the IT-framework as needed using a low-code approach. If you have a PWA (Progressive Web Apps) online shop or custom web interfaces and apps for your sales teams, you can quickly implement and improve any commerce solution. 
+
+
+### Flexible APIs
+
+Companies can accelerate innovation with modern APIs (Advanced Programming Interfaces) and easy-to-use low-code development environment. With 500+ API endpoints, you can integrate existing backend systems or additional third-party applications. You can extend the platform to your needs &ndash; regardless of the front end of your customer's devices. 
+
+
+### Cloud technology
+
+Instead of worrying about maintaining, connecting, and updating a monolithic ERP software, you can use a fail-safe and scalable platform made in Germany. Our cloud servers are compliant with the European General Data Protection Regulation (GDPR). They are located in Germany.
```

Comment:
Vorschlag:  Our cloud servers are compliant with the European General Data Protection Regulation (GDPR), as they are located in Germany. (oder As they are located in Germany, our cloud servers...)